packages/codeeditor/src/editor.ts

```diff
@@ -451,10 +451,18 @@ namespace CodeEditor {
 
     /**
      * Repaint the editor.
+     * 
+     * #### Notes
+     * A repainted editor should fit to its host node.
      */
     refresh(): void;
 
     /**
+     * Resize the editor to fit its host node.
+     */
+    resizeToFit(): void;
```

Comment:
@blink1073 Could you take care about code mirror implementation that i don't overlook something? Calling only `refresh` will do with monaco.